var/spack/repos/builtin/packages/libpciaccess/package.py

```diff
@@ -28,3 +30,15 @@ class Libpciaccess(AutotoolsPackage, XorgPackage):
     # When the ability to use dependencies built by another compiler, using a
     # libpciaccess built by gcc should be usable by PGI builds.
     conflicts('%pgi')
```

Comment:
Wait, I've been having issues building libpciaccess with PGI for years. Could a similar fix be used to get PGI working?

Reply:
Probably, although I have not tested it.  Of course now you should be using the HPC SDK instead of the PGI anyway.  :)